docs/source/getting-started/security-basics.rst

```diff
@@ -80,6 +80,44 @@ To achieve this, simply omit the configuration settings
 ``c.JupyterHub.ssl_key`` and ``c.JupyterHub.ssl_cert``
 (setting them to ``None`` does not have the same effect, and is an error).
 
+.. _authentication-token:
+
+Proxy authentication token
+--------------------------
+
+The Hub authenticates its requests to the Proxy using a secret token that
+the Hub and Proxy agree upon. The value of this token should be a random
+string (for example, generated by ``openssl rand -hex 32``).
+
+Generating and storing token in the configuration file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Or you can set the value in the configuration file, ``jupyterhub_config.py``:
```

Comment:
Should there be an "or" here? Makes me think we lost the alternative option

Reply:
Ah, reading on I think I see the second option :) Maybe we can put a sentence like "You can store it in the configuration file or an environment variable"  on line 90. Then remove the "Or" here.

Reply:
Sounds good to me. Done!